optimism.tokenlist.json

```diff
@@ -1569,6 +1569,39 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+    {
+      "chainId": 1,
+      "address": "0x3258cd8134b6b28e814772dd91d5ecceea512818",
+      "name": "Land",
+      "symbol": "LAND",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/LAND.png",
+      "extensions": {
```

Comment:
If you do not plan on bridging this token from mainnet to Optimism mainnet, please remove the extension.